ui/src/app/pages/admin/workspace/admin-workspace.tsx

```diff
@@ -63,23 +39,23 @@ export class AdminWorkspaceImpl extends React.Component<Props, State> {
     this.state = {
       workspaceDetails: {},
       cloudStorageTraffic: null,
-      loadingWorkspaceAdminLockedStatus: false,
-      showLockWorkspaceModal: false,
     };
   }
 
   componentDidMount() {
     this.props.hideSpinner();
-    this.getFederatedWorkspaceInformation();
+    this.populateFederatedWorkspaceInformation();
   }
 
-  componentDidUpdate(prevProps) {
-    if (hasNewValidProps(this.props, prevProps, [(p) => p.match.params])) {
```

Comment:
this was a silent bug: `hasNewValidProps()` does not actually work on objects.  I have updated the type signature to reflect this: it is now an error to call it in this way.